stable/anchore-admission-controller/README.md

```diff
@@ -39,7 +39,7 @@ Not all users in the anchore engine need to be specified, only those that will b
 To create the secret:
 
 ```
-kubectl create secret generic anchore-credentials --from-file=creds.json=credentials.json
+kubectl create secret generic anchore-credentials --from-file=creds.json --from-file=credentials.json --from-file=auth.json
```

Comment:
I think this should be just:
kubectl create secret generic anchore-credentials --from-file=credentials.json=credentials.json

That assumes that credentials.json is a json with structure outlined in the rest of the readme.

Reply:
Hi @zhill I've just tried with kubectl create secret generic anchore-credentials --from-file=creds.json=credentials.json which fails.

kubectl create secret generic anchore-credentials --from-file=credentials.json works.

Reply:
yes, agreed. I'm asking about the `--from-file=auth.json`. I'm not sure where that is coming from and shouldn't be necessary in the readme. If you can remove that then I'll merge this. Thanks again!

Reply:
Hi @zhill , yes just the following...

> kubectl create secret generic anchore-credentials --from-file=creds.json --from-file=credentials.json